website/content/en/docs/building-operators/helm/migration.md

```diff
@@ -74,12 +73,64 @@ $ operator-sdk create api \
  
-### Replacing the content
+### Migrating your Custom Resource samples
 
-- Update the CR manifests in `config/samples` with the values of the CR's in your old project which are in `deploy/crds/`
-- Check if you have customizations options in the `watch.yaml` file of your previous project and then, update the new `watch.yaml` file with the same ones
-- Ensure that all roles configured in the `/deploy/roles.yaml` will be applied in the new project in the file `config/rbac/role.yaml`
-- If you have customizations in your `helm-charts` then, apply them in the new `helm-charts`. Note that this directory was not changed at all.
+Update the CR manifests in `config/samples` with the values of the CRs in your existing project which are in `deploy/crds/<group>_<version>_<kind>_cr.yaml`
+
+### Migrating `watches.yaml`
+
+Check if you have custom options in the `watches.yaml` file of your existing project. If so, update the new `watches.yaml` file to match. In our example, it will look like:
+
+```yaml
+# Use the 'create api' subcommand to add watches to this file.
+- group: example.com
+  version: v1alpha1
+  kind: Nginx
+  chart: helm-charts/nginx
+# +kubebuilder:scaffold:watch
+```
+
+**NOTE**: Do not remove the `+kubebuilder:scaffold:watch` [marker][marker]. It allows the tool to update the watches file when new APIs are created. 
+
+### Checking the Permissions (RBAC)
+
+In your new project, roles are automatically generated in `config/rbac/role.yaml`.
+If you modified these permissions manually in `deploy/role.yaml` in your existing
+project, you need to re-apply them in `config/rbac/role.yaml`.
+
+New projects are configured to watch all namespaces by default, so they need a `ClusterRole` to have the necessary permissions. Ensure that `config/rbac/role.yaml` remains a `ClusterRole` if you want to retain the default behavior of the new project conventions. For further information refer to the [operator scope][operator-scope] documentation.  
+
+The following rules were used in earlier versions of helm-operator to automatically create and manage services and servicemonitors for metrics collection. If your operator's charts don't require these rules, they can safely be left out of the new `config/rbac/role.yaml` file:
+
+```  
+  - apiGroups:
+    - monitoring.coreos.com
+    resources:
+    - servicemonitors
+    verbs:
+    - get
+    - create
+  - apiGroups:
+    - apps
+    resourceNames:
+    - memcached-operator
+    resources:
+    - deployments/finalizers
+    verbs:
+    - update
+```
+
+### Migrating your Helm Charts
+
+Above, you already created the API with the Helm-Chart using the command `operator-sdk create api`. However, if you have in your previous project customizations which were applied just in the operator and NOT in the `Helm-chart` used to create the API then, ensure that you will migrate them to the new project as well. 
```

Comment:
I still disagree with this section. If I run `operator-sdk create api --helm-chart=<existing_project>/helm-charts/<chart>`, as is documented earlier in this doc, there will be no further helm chart migrations necessary.

The `./helm-charts` dirs in both old and new projects will be identical.

Reply:
Using a directory path with the `--helm-chart` flag effectively just copies the directory into the new project.

Reply:
Yep. It is true.